ui/apps/dashboard/src/components/Metrics/ChartTooltip.tsx

```diff
@@ -0,0 +1,26 @@
+import type { DefaultLabelFormatterCallbackParams } from '@inngest/components/Chart/Chart.jsx';
```

Comment:
You probably need to remove the .jsx from that import, I'm assuming that happened thanks to the autocomplete

Reply:
Nice catch, fixed.